runtime/src/main/protobuf/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
This file is not needed for server reflection

Reply:
It is needed to generate `AnnotationsProto` which is used at https://github.com/akka/akka-grpc/pull/654/files#diff-55da3d8b2cdb4c2612a6d74c31c31ee5R55 . It might be possible to load it from a dependency instead of including the .proto here, I suppose?